lang/luajit/Makefile

```diff
@@ -0,0 +1,91 @@
+include $(TOPDIR)/rules.mk
```

Comment:
@diizzyy 
You can just omit it. Just like said in your link ...

> or omit it entirely.

The whole copyright discussion is out of hand.